File: 2 SEGUNDO/1 Cuatrimestre/Programación y Diseño Orientado a Objetos - PDOO/Practicas/rubyNapakalaki/EjemploMain.rb
require_relative "napakalaki.rb"
require_relative "GameTester.rb"

module NapakalakiGame

  class EjemploMain

      def prueba

       test = Test::GameTester.instance

       game = Napakalaki.instance

       test.play(game, 2);

      end

  end

    e = EjemploMain.new
    e.prueba()

end

File: 2 SEGUNDO/1 Cuatrimestre/Programación y Diseño Orientado a Objetos - PDOO/Practicas/javaNapakalaki/NapakalakiGame/EjemploMain.java
package NapakalakiGame;

import Test.GameTester;

public class EjemploMain {

    public static void main(String[] args) {
      Napakalaki game = Napakalaki.getInstance();
      GameTester test = GameTester.getInstance();

      test.play(game, 2);

    }
}
